charts/handbrake/0.0.1/values.yaml

```diff
@@ -0,0 +1,91 @@
+image:
```

Comment:
Same comments as for the test_values.yaml also apply here, in case you wonder...